extension/task/task.json

```diff
@@ -11,21 +11,26 @@
   "author": "Tingle Software",
   "demands": ["docker"],
   "version": {
-    "Major": 1,
-    "Minor": 5,
+    "Major": 2,
+    "Minor": 0,
```

Comment:
@mburumaxwell 
I'm not sure what the implications for this are, but I figured it was probably best to separate the tasks by a major version since the options are quiet different.

My understanding is that users should be able to switch between v1 and v2 if for whatever reason v2 causes problems.

Let me know if you have any concerns about this or if you want me to dig in to it some more